docs/development/development.md

```diff
@@ -125,6 +125,13 @@ If you're unable to solve the merge conflicts, don't worry you'll still be able
 
 Give your PR a meaningful title and provide details about the change in the description, including a link to the issue(s) relating to your PR. All that's left is to click the 'Create pull request' button and wait for our eager review of your code!
 
+### Python Style guide
+
+* Use 4 spaces for a tab.
```

Comment:
Put this into the Coding Conventions section

Reply:
There is an outdated section "Coding Conventions" that was probably relevant to bash in the previous BitOps v1:
https://bitovi.github.io/bitops/contributing/#coding-conventions
That section was removed in this PR.

There is another section for `bash` and `yaml` formatting under the development page:
https://bitovi.github.io/bitops/development/development/#bash-style-guide

I propose to have only one place for the coding standards under development.md page to avoid confusion with scattered guidelines.